src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,50 +1,223 @@
+        printTable(tableName);
 
+        removeUserFromTable(tableName, "user3");
+        printTable(tableName);
+
+
+        if (connection != null) {
+            try {
+                connection.close();
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
         }
+    }
 
-        System.out.println("PostgreSQL JDBC Driver Registered!");
+    private void printStatusMsg(String operation, String subject) {
+        System.out.println(operation + " table " + subject + "..");
+    }
+
+    private void dropTableifExists(String tableName) {
+        String sqlQuery = "DROP TABLE IF EXISTS " + tableNameFormatted(tableName);
+        printStatusMsg("Removing", tableName);
+        int affectedRows = executeQuery(sqlQuery);
+        printMsgAffectedRows(affectedRows);
+    }
 
-        Connection connection = null;
+    private void changePasswordInTable(String tableName, String name, String password) {
 
-        try {
+        String sqlQuery = "UPDATE " + tableNameFormatted(tableName) + " SET password = '" + password + "' WHERE name = '" + name + "'";
+        printStatusMsg("Updating", tableName);
+        int affectedRows = executeQuery(sqlQuery);
+        printMsgAffectedRows(affectedRows);
+    }
+
+    private void removeUserFromTable(String tableName, String value) {
+        String sqlQuery = String.format("DELETE FROM " + tableNameFormatted(tableName) + " WHERE name='%s'", value);
+        int affectedRows = executeQuery(sqlQuery);
+        printMsgAffectedRows(affectedRows);
+    }
+
+    private String tableNameFormatted(String tableName) {
+        return "\"" + tableName + "\"";
+    }
+
+
+    private void printTable(String tableName) {
+        String sqlQuery = "SELECT * FROM " + tableNameFormatted(tableName);
+        try (PreparedStatement statement = connection.prepareStatement(sqlQuery)) {
+            printStatusMsg("Printing", tableName);
+            statement.execute();
+            ResultSet resultSet = statement.getResultSet();
+
+            if (!resultSet.isBeforeFirst()) {
+                System.out.println("table is empty");
+                System.out.println();
+                return;
+            }
+
+            StringBuilder stringBuilder = new StringBuilder();
+            int columnsCount = getColumnsCountFrom(resultSet);
+            while (resultSet.next()) {
+                String columnsSeparator = " |";
+                for (int i = 1; i <= columnsCount; i++) {
+                    stringBuilder.append(resultSet.getString(i));
+                    stringBuilder.append(columnsSeparator);
+                }
+                truncate(stringBuilder, columnsSeparator.length());
+                stringBuilder.append(LINE_SEPARATOR);
+            }
+
+            System.out.println(stringBuilder);
+
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void truncate(StringBuilder stringBuilder, int charsCount) {
```

Comment:
I thought originally this is method for Truncate table:) Why you use separate method instead of use line
stringBuilder.delete(stringBuilder.length() - charsCount, stringBuilder.length()); 
in cycle?

Reply:
Ok, I’ll check it tonight

Reply:
Originally it was intended for readability: it is better to read method's name than awkward line of code. I don't like write comments neither. But I didn't see that method contained only 1 line of code)